apps/calendar/src/factory/calendarControl.tsx

```diff
@@ -177,7 +177,29 @@ export default abstract class CalendarControl implements EventBus<ExternalEventT
     };
   }
 
-  private move(offset = 0) {
+  /**
+   * Move the rendered date to the next/prev range.
+   *
+   * The range of movement differs depending on the current view, Basically:
+   *   - In month view, it moves to the next/prev month.
+   *   - In week view, it moves to the next/prev week.
+   *   - In day view, it moves to the next/prev day.
+   *
+   * Also, the range depends on the options like how many visible weeks/months should be rendered.
+   *
+   * @param offset{number} The offset to move by.
```

Comment:
```suggestion
   * @param {number} offset - The offset to move by.
```

jsdoc 규칙에 따르면 타입 먼저 오고 그 뒤 네임이 와야 해요.
* 참고: https://jsdoc.app/tags-param.html